system/jlib/jstats.cpp

```diff
@@ -1224,6 +1226,34 @@ void StatisticsMapping::createMappings()
         unsigned kind = indexToKind.item(i2);
         kindToIndex.replace(i2, kind);
     }
+
+    const unsigned * kinds = indexToKind.getArray();
+    hashcode = hashc((const byte *)kinds, indexToKind.ordinality() * sizeof(unsigned), 0x811C9DC5);
+
+    //All StatisticsMapping objects are assumed to be static, and never destroyed.
```

Comment:
because assumed to be static objects, we can assume use of allStatsMappings is thread safe..
can be be sure future cases will always be via static objects though?
Would it be safer to add an initialization mutex, and not significant enough overhead to worry about?